nx-X11/programs/Xserver/Xext/panoramiX.c

```diff
@@ -129,11 +130,11 @@ static void XineramaValidateGC(GCPtr, unsigned long, DrawablePtr);
 static void XineramaChangeGC(GCPtr, unsigned long);
 static void XineramaCopyGC(GCPtr, unsigned long, GCPtr);
 static void XineramaDestroyGC(GCPtr);
-static void XineramaChangeClip(GCPtr, int, void *, int);
+static void XineramaChangeClip(GCPtr, int, pointer, int);
```

Comment:
We don't want to re-introduce ``pointer`` here. Keep ``void *`` here.

Reply:
fixed